ports/battalion/port.json

```
@@ -0,0 +1,41 @@
+{
+    "version": 3,
+    "name": "battalion.zip",
+    "items": [
+        "BATTALION.sh",
+        "battalion/"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "BATTALION ",
+        "porter": [
+            "MadShmupper"
+        ],
+        "desc": "Your lands have been invaded by General Mustache and his infamous armored division!\nNothing seems to stop him, except perhaps for Commander Joe and his old companion \"G-080 Justice\" tank destroyer.\nEngage in 8 exciting battles through the liberation campaign.\nFight in cities, forests, riverbanks, military bases and more...\nCollect the powerup to experience ultimate defense and crush obstacles on the way.",
+        "desc_md": null,
+        "inst": "READY to RUN ",
+        "inst_md": null,
+        "genres": [
+            "action",
+            "strategy"
+        ],
+        "image": null,
+        "rtr": true,
+        "exp": false,
+        "runtime": null,
+        "reqs": [],
+        "arch": [
+            "armhf"
+        ]
+    },
+    "status": {
+        "source": "Unknown",
+        "md5": null,
+        "status": "Installed"
+    },
+    "files": {
```

You have these entries created by harbourmaster in your port.json
```suggestion
    "files": {
```